### PR TITLE
Add page on Surface Albedo

### PR DESCRIPTION
**Pull request by kanderso-nrel**
This PR adds a new page on some interesting (and perhaps unexpected) aspects of PSM3's Surface Albedo data.

It also begins a citations page so that NSRDB publications can be referenced easily.  Since we're using nbsphinx, we can't use normal RST citation roles, so instead we have to use a strange HTML syntax described here: https://nbsphinx.readthedocs.io/en/0.8.12/markdown-cells.html#Citations

Good enough for now, but maybe we can find a better way in the future.

### CODE REVIEW

File: source/pages/albedo.ipynb [hunk 1]
```diff
@@ -0,0 +1,358 @@
+{
```
Comment (AdamRJensen) [inline source/pages/albedo.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    pvlib.iotools.psm3.PSM_URL = "https://developer.nrel.gov/api/nsrdb/v2/solar/psm3-2-2-download.csv"
</pre><p>Perhaps add a comment as to why you are changing the base URL?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (AdamRJensen) [inline source/pages/albedo.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.        df, meta = pvlib.iotools.get_psm3(47.42, -88.05, 'DEMO_KEY', 'assessingsolar@gmail.com',
</pre><p>Would be nice to have each parameter on a single line and add the argument name, e..g,</p><p> </p><p>    df, meta = pvlib.iotools.get_psm3(</p><p>        latitude=47.42,</p><p>        longitude,-88.05,</p><p>        ...</p><p>    )</p><p> </p><p>and so forth</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (AdamRJensen) [inline source/pages/albedo.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    plt.ylabel('Surface Albedo')
</pre><p> </p><p>I like adding [-] just to denote that it is unitless and be consistent with some of the other plots where you include the square bracket and units if it's temperature. I don't feel very strongly though, so feel free to ignore.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (AdamRJensen) [inline source/pages/albedo.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.    ax.set_ylabel('PSM3 Albedo')
</pre><p>I guess the y-label should just be <code>'Albedo'</code> to be consistent with the remaining plots.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (AdamRJensen) [inline source/pages/albedo.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    filt = daily_albedo &gt; 0.6
</pre><p>Let's just call the variable <code>snow_filter</code> or <code>snow_bool</code> and not be lazy ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (AdamRJensen) [inline source/pages/albedo.ipynb:1]: <p>Do you know why there is the staircase like behavior? Due to lack of decimals?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

**Review by AdamRJensen — APPROVED** [review]
As usual, very well done. I posted a few comments using reviewnb that you can check out.

File: source/pages/albedo.ipynb (hunk 1 above)
Comment (kanderso-nrel) [reply source/pages/albedo.ipynb:1]: Yep, the API returns albedo values rounded to two decimal places

**Pull request merged**